easyInterface/Diffraction/Interface.py

```diff
@@ -397,7 +413,7 @@ def updateExperiments(self) -> NoReturn:
             self.project_dict.setItemByPath(['info', 'experiment_ids'], list(experiments.keys()))
             self.project_dict.endBulkUpdate()
         else:
-            k, v = self.project_dict['experiments'].dictComparison(experiments)
+            k, v, t = self.project_dict['experiments'].dictComparison(experiments)
```

Comment:
Should we use a less cryptic names, such as `keys`, `values`? And what is `t`? It seems that `t` is not used.